Tests/Core/AWSClientRuntimeTests/Sigv4/SigV4SigningTests.swift

```diff
@@ -9,23 +9,13 @@ import AwsCommonRuntimeKit
 import ClientRuntime
 import XCTest
 
-@testable import AWSClientRuntime
+@testable @_spi(Internal) import AWSClientRuntime
```

Comment:
What SPI is this intended to access?  (I thought we had renamed the SPI that was previously named `Internal`)

Reply:
On line 70, the file based config line `try! await credentialsProvider.configure(.init(fileBasedConfigurationStore: .init()))`